File: 전예진/BruteForce/prob2.py
def solution(answers):
    answer = []
    
    a = [1, 2, 3, 4, 5]
    b = [2, 1, 2, 3, 2, 4, 2, 5]
    c = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5]
    cnt = [0, 0, 0]
    
    for i in range(len(answers)):
        ans = answers[i]
        
        if a[i % len(a)] == ans:
            cnt[0] += 1
        
        if b[i % len(b)] == ans:
            cnt[1] += 1
        
        if c[i % len(c)] == ans:
            cnt[2] += 1
        
      
    # 최대값 구하기
    max_value = max(cnt)
    for i in range(3):
        if max_value == cnt[i]:
            answer.append(i+1)
    
    return answer

File: 주다애/0325/prob2.java
class Solution {
    public int[] solution(int[] prices) {
        int len = prices.length;
        int[] answer = new int[len];
        int cnt = 0;
        for(int i = 0; i < len - 1; i++) {
            for(int j = i + 1; j < len; j++) {
                answer[i]++;
                if(prices[i] > prices[j]) {
                    break;
                }
            }
        }
        return answer;
    }
}
